### ISSUE TITLE
Troube starting RocketChat in Internet Explorer

### ISSUE THREAD

**Issue by andytipp** [RocketChat/Rocket.Chat#8204]
Using IE 11 on Windows 7, the RocketChat login screen won't load on demo.rocket.chat

![rocketchat bug](https://user-images.githubusercontent.com/23705040/30581723-735005a8-9d21-11e7-8fa5-d14945522be5.PNG)
![rocketchat bug2](https://user-images.githubusercontent.com/23705040/30581908-187d1354-9d22-11e7-9621-99ebbde100b3.PNG)

**Comment by mrsimpson** [comment]
@RocketChatApp in 0.57.2, this issue did not occur.

**Comment by mrsimpson** [comment]
@RocketChat/core
The side panel (flextab) not opening in 0.58.x is due to a flex-property.
See https://github.com/assistify/Rocket.Chat/pull/78#discussion_r143340198 for details.
I consider this a really severe bug as a lot of users are still on IE11 and would not be able to use RocketChat with the new UI based on the current design!

**Comment by apitt** [comment]
Yes.  We're stuck at 0.57.3 because of this.  We have about 500 users, but IE11 is still our corporate standard (for now).
